R/get_parsing_tables.R

```diff
@@ -98,13 +98,17 @@ getParsingTables = function(tableSuffix) {
 
 
 createParsingTables = function(
-  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL,
+  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL, tableNames = NULL,
   ...) {
   if (is.null(dbname)) return(invisible())
 
   con = connect(dbtype, dbname, ...)
   parTables = getParsingTables(tableSuffix)
 
+  if(!is.null(tableNames) && length(tableNames > 0)){
```

Comment:
- whitespace
- Why does the if statement need the `&& length(...`?

Reply:
I just want to make sure an empty list isn't passed as an extra precaution, but if you don't think it's necessary then I'll just remove it!